test/functional/terminal/tui_spec.lua

```diff
@@ -1529,119 +1529,44 @@ describe("TUI", function()
-    {5:[No Name]                       0,0-1          All}|
-                                                      |
-    {3:-- TERMINAL --}                                    |
-    ]])
-    feed_data('\027]11;rgba:ffff/ffff/ffff/8000\027\\')
-    screen:expect{any='did OptionSet, yay!'}
+  before_each(clear)
 
-    feed_data(':echo "new_bg=".&background\n')
-    screen:expect{any='new_bg=light'}
+  it('is properly set in a nested Neovim instance when background=dark', function()
+    command('set background=dark')  -- set outer Neovim background
+    local screen = thelpers.screen_setup(0, string.format(
+      '["%s", "-u", "NONE", "-i", "NONE", "--cmd", "set noswapfile"]', nvim_prog))
+    retry(nil, 30000, function()  -- wait for automatic background processing
+      screen:sleep(20)
+      feed_command('set background?')  -- check nested Neovim background
+      screen:expect([[
+      {1: }                                                 |
+      {4:~                                                 }|
+      {4:~                                                 }|
+      {4:~                                                 }|
+      {5:[No Name]                       0,0-1          All}|
+        background=dark                                 |
+      {3:-- TERMINAL --}                                    |
+      ]])
+    end)
   end)
 
-  it('triggers OptionSet event with split terminal-response', function()
-    screen:expect([[
-    {1: }                                                 |
-    {4:~                                                 }|
-    {4:~                                                 }|
-    {4:~                                                 }|
-    {5:[No Name]                       0,0-1          All}|
-                                                      |
-    {3:-- TERMINAL --}                                    |
-    ]])
-    -- Send a background response with the OSC command part split.
-    feed_data('\027]11;rgb')
-    feed_data(':ffff/ffff/ffff\027\\')
-    screen:expect{any='did OptionSet, yay!'}
-
-    feed_data(':echo "new_bg=".&background\n')
-    screen:expect{any='new_bg=light'}
-
-    setup()
-    screen:expect([[
-    {1: }                                                 |
-    {4:~                                                 }|
-    {4:~                                                 }|
-    {4:~                                                 }|
-    {5:[No Name]                       0,0-1          All}|
-                                                      |
-    {3:-- TERMINAL --}                                    |
-    ]])
-    -- Send a background response with the Pt portion split.
-    feed_data('\027]11;rgba:ffff/fff')
-    feed_data('f/ffff/8000\007')
-    screen:expect{any='did OptionSet, yay!'}
-
-    feed_data(':echo "new_bg=".&background\n')
-    screen:expect{any='new_bg=light'}
+  it('is properly set in a nested Neovim instance when background=light', function()
```

Comment:
"neovim" is the project name. Nvim is the editor name. nvim is the cli name.
```suggestion
  it('is properly set in a nested Nvim instance when background=light', function()
```

Reply:
Done (f30cc5376df62449956f9700afa903c689c7986b).